pkg/sundress/kms.go

```diff
@@ -0,0 +1,70 @@
+package sundress
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/kms"
+	"log"
+)
+
+type secret struct {
+	keyId  *string
+	client *kms.KMS
+}
+
+var (
+	sec *secret
+)
+
+func init() {
+	sec = getSecret()
+}
+
+func getSecret() *secret {
+	//need to change how credentials are stored, rn using my local
+	sess, err := session.NewSession(&aws.Config{
+		Region: aws.String("us-east-1")},
+	)
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	svc := kms.New(sess)
+
+	//need to replace the key with bdc stuff
+	return &secret{
+		keyId:  aws.String("arn:aws:kms:us-east-1:460622542287:key/8b4b0689-b1f2-4fb0-90cd-11f8a0c080ca"),
+		client: svc,
+	}
+}
+
+func Encrypt(s string) string {
+	if true {
```

Comment:
lol just gonna merge this now, cuz i wanna add some more routes to this tn - just make a follow up PR or sm after